docs/docsite/rst/porting_guide_2.0.rst

```diff
@@ -1,9 +1,19 @@
-Porting Guide
-=============
+.. _porting_2.0_guide:
 
+*************************
+Ansible 2.0 Porting Guide
+*************************
+
+This section discusses the changes between Ansible 1.x and 2.0.
+
+It is intended to assist in updating your playbooks, plugins, etc so they will work with this version of Ansible.
```

Comment:
Please avoid using "etc". Suggest changing to either simply "playbooks and plugins" or maybe "your Ansible infrastructure".